config.json

```diff
@@ -61,6 +61,13 @@
       ]
     },
     {
+      "slug": "acronym",
+      "difficulty": 2,
+      "topics": [
+        "String Manipulation"
```

Comment:
I was thinking whether we could find a way to classify exercises based on student feedback.
I'm gonna give a suggestion here first because you have far more experience with Exercism than I do, so you may be aware of reasons for this idea to not work.

Say, for instance, the student could rate the exercise in the submission page. We could ask for feedback in two opportunities:

1. In the submission page, the User finds an option to rate the difficulty of the exercise; or
1. Upon archiving the exercise, the User is prompted with a request to rate the difficulty exercise.

I favour option **1** due to many Users (including me) not archiving their exercises.

Then, we could use their feedback to help us sort the exercises on a given period (say, once every 2 months).
Certainly, there is a high degree of subjectivity with this system. A Student already familiar with the Language could rate an exercise `Very Easy`, while beginners would rate it `Hard`. Therefore, maybe we could set some sort of threshold. For example, if we get more than 40% of the feedback suggesting the Exercise is far off the current level, than we should reorder it.

This system could also help us improve Exercism in more than one way, for example:

1. Discover what topics the students are having more difficulty in a Track, then try to add more exercises on that topic;
1. Find _holes_ in a Track, such as students finding an exercise much more difficult than the previous one, so that we can add more exercises in between.
1. Find _overloaded_ topics, such as students rating successive exercises easier, which could mean we are adding too many exercises on that topic, causing students to get bored.

I apologise for often giving such big comments/replies, I hope they are constructive to Exercism in some way :-)